dev/docker/ruby2.5/docker-compose.run.yml

```diff
@@ -1,4 +1,4 @@
 version: '3'
 services:
   ruby:
-    command: ['bash', '-c', 'bundle install --deployment;sleep 10;RUBY_ENV=magento2 bundle exec rake project:anonymize[magento2]']
\ No newline at end of file
+    command: ['bash', '-c', 'bundle install --deployment;sleep 10;RUBY_ENV=magento2 bundle exec rake project:anonymize[${MAGENTO_VERSION}]']
```

Comment:
since the env config is the same, we should change RUBY_ENV to magento and rename the file /config/env/magento2.yml